context-menu-demo/README.md

```diff
@@ -0,0 +1,27 @@
+# context-menu-demo
+
+A demo of the [contextMenus API](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/contextMenus/).
+
+## What it does
+
+This add-on adds several items to the browser's context menu:
+
+* one shown when there is a selection in the page, that logs the selected text
+when clicked.
+* one shown in all contexts, that is removed when clicked.
+* two "radio" items that are shown in all contexts.
+These items are grouped using a separator item on each side.
+One radio item adds a blue border to the page, the other adds a green border.
```

Comment:
minor point that these don't work on about:debugging, which is the first page I load it from, not sure that's a biggie though